Harmonise wording for techniques checks/steps

patrickhlauke opened the pull request:
Currently, techniques use a mix of wording `If step X...` and `If check X...`, as well as a few outlier "If test procedure X...". Most prefix the step/check number with `#`, but some don't. And conceptually, I think it makes more sense saying that a *check* is true/false, rather than a *step* being true/false.

This PR tries to harmonise this, settling for `check` and adding missing `#` (though, it has to be said, this use of the octothorpe is a bit US-centric, so maybe we want to harmonise to *not* having it?)

patrickhlauke committed:
Harmonise wording for techniques checks/steps

patrickhlauke committed:
Add outlier "If test procedure"

patrickhlauke committed:
Missing `#`

fstrr commented:
Agree on the US-centric use of the `#` symbol. I've often considered yoinking it out and would be fine with it being removed.

patrickhlauke commented:
Point to discuss in our next WCAG 2.x meeting then ... if we want to yoink `#` out, I can do a big find/replace and add it as a commit here.

alastc commented:
I think aligning with the earlier examples is probably best (and the least amount of files changed), which is what you've done. That will also ruffle the least feathers.

mgifford commented on techniques/failures/F52.html:
I think this is better as check #2 and check #3 but in other places it is checks #2 and #3. Just wanted to highlight this.

mgifford commented on techniques/failures/F9.html:
This is inconsistent. Should be all checks and no steps, right?

ljoakley committed:
Update F38.html

changing "check #1 is true and check #2 is true" to match the rest of changes: "check #1 and #2 is true"

ljoakley committed:
Update F61.html

removed the word "both" as none of the other changes use this

ljoakley reviewed (approved):
completed with a few changes

patrickhlauke commented on techniques/failures/F24.html:
```diff
@@ -90,7 +90,7 @@ <h3>Specifying foreground color of link text with CSS</h3>
       </section>
       <section class="results"><h3>Expected Results</h3>
-         <p>If step #2 is true but step #3 is false, OR if step #3 is true but step #2 is false then this
+         <p>If check #2 is true but step #3 is false, OR If check #3 is true but step #2 is false then this
```
```suggestion
         <p>If check #2 is true but step #3 is false, <em>or</em> if check #3 is true but step #2 is false then this
```

patrickhlauke committed:
Update techniques/failures/F24.html

patrickhlauke commented on techniques/failures/F38.html:
```diff
@@ -22,7 +22,7 @@
       <section class="results"><h3>Expected Results</h3>
          <ul>
-            <li>If step #1 is true and if step #2 is true, this failure condition applies and content fails the Success Criterion.</li>
+            <li>If check #1 and #2 is true, this failure condition applies and content fails the Success Criterion.</li>
```
```suggestion
            <li>If check #1 and #2 are true, this failure condition applies and content fails the Success Criterion.</li>
```

patrickhlauke committed:
Update techniques/failures/F38.html

mbgower commented:
@shawna-slh Is there any style decision on use of the pound symbol (#) in front of ordered list references?

shawna-slh commented:
Thanks for the ping @mbgower ! We don't currently have that in our Style guide. (I prefer without the # - yet don't have any support for or against that.) From a quick search, I don't find guidance in leading Style Guides -- so it needs more research.

The good news is we have a new WAI staff person coming on the end of the month who will be great at citing sources for a decision.

Are y'all comfortable leaving this issue open for a few weeks until we can get back to it?

(p.s. thanks to @patrickhlauke I learned a new term, "octothorpe".)

patrickhlauke commented:
> Are y'all comfortable leaving this issue open for a few weeks until we can get back to it?

no rush from my perspective. I'd rather wait and then do all the changes in a oner, once we have a decision on the `#`

mbgower commented:
@shawna-slh Sorry, I missed those final comments when I brought this into our current review cycle.
If someone gets to this by the start of August, we can just incorporate your change, otherwise we'll just push a second update when you make your recommendation.

shawna-slh commented:
@tamsinewing555 will probably get to it within a few days :)

mbgower committed:
Merge branch 'main' into patrickhlauke-techniques-step-fail-language-tweak
